automation/src/test/scala/org/broadinstitute/dsde/test/api/RawlsConfig.scala

```diff
@@ -5,4 +5,12 @@ import org.broadinstitute.dsde.workbench.config.CommonConfig
 object RawlsConfig extends CommonConfig {
   private val gcs = config.getConfig("gcs")
   val pathToQAJson = gcs.getString("qaJsonFile")
+
+  private val fireCloud = config.getConfig("fireCloud")
+  // This value is only available when tests are run through the rawls-swat Github action (in a bee).
+  val wsmUrl = if (fireCloud.hasPath("workspaceManagerApiUrl")) {
+    fireCloud.getString("workspaceManagerApiUrl")
+  } else {
+    "ERROR: workspaceManagerApiUrl unspecified! "
```

Comment:
This is how the error would present if you try to run the tests with the config option missing (such as Jenkins swatomation job):

```
ERROR: workspaceManagerApiUrl unspecified! status (of class java.lang.String)
scala.MatchError: ERROR: workspaceManagerApiUrl unspecified! status (of class java.lang.String)
	at org.broadinstitute.dsde.workbench.service.RestClient.toUri$1(RestClient.scala:42)